.buildkite/pipelines/pmtile_synth_only.yml

```diff
@@ -19,15 +22,14 @@ steps:
 # INDIVIDUAL TILE RUNS
 # Builds in dir e.g. mflowgen/full_chip/19-tile_array/16-Tile_MemCore
 
-- label: '250MHz PE synth 12m'
+- label: 'PE synth 20m'
   commands:
   - $TEST --need_space 30G full_chip tile_array Tile_PE --steps synthesis --debug
   - .buildkite/pipelines/check_pe_area.sh
   - mflowgen/bin/buildcheck.sh full_chip/*tile_array/*Tile_PE --show-all-errors
-- wait: { continue_on_failure: true } # One step at a time + continue on failure
 
-- label: 'MemCore synth 17m'
+- label: 'MemCore synth 25m'
```

Comment:
(The runtime for these two tests was off, it's not necessarily related to these latest code changes.)